imap/promstatsd.c

```diff
@@ -588,43 +630,58 @@ static void do_collate_usage_report(struct buf *buf)
     int64_t starttime;
 
     buf_reset(buf);
-    construct_hash_table(&h, 10, 0); /* 10 partitions is probably enough right */
+    construct_hash_table(
+        &h, 10, 0); /* 10 partitions is probably enough right */
```

Comment:
I wonder if it could not wrap when the only reason it needs to wrap is a comment

Reply:
... or wrap the comment